cypress/TestCases/FireboltCertification/Account.feature

```diff
@@ -4,24 +4,24 @@ Feature: Account
    Background: Launch FCA for 'Account'
       Given the environment has been set up for 'Account' tests
       And 3rd party 'certification' app is launched
-
-   @sdk @transport
-   Scenario Outline: Account.<Method> - Positive Scenario: <Scenario>
+#
+   @sdk @transport @Sev0
+   Scenario Outline: <Scenario> Account.<Method>
```

Comment:
Format here is typically <module>.<api> - <descriptive name>

ex: Account.uid - Validate method response content